releasenotes/notes/1.3/oxidize-transpiler-passes-16f2f1c57612e74a.yaml

```diff
@@ -0,0 +1,53 @@
+---
+features_transpiler:
+  - |
+    Most of the transpiler passes required by the preset pass manager have
+    been ported into Rust.
+
+    * The :class:`.ConsolidateBlocks` pass will now run the equivalent of the
+      :class:`.Collect2qBlocks` pass internally if it was not run in a pass
+      manager prior to the pass. Previously, 
+      :class:`.Collect2qBlocks` or :class:`.Collect1qRuns` had to be run before
+      :class:`.ConsolidateBlocks` for :class:`.ConsolidateBlocks` to do
+      anything. By doing the collection internally, the overhead from the pass
+      is reduced. If :class:`.Collect2qBlocks` or :class:`.Collect1qRuns` are
+      run prior to :class:`.ConsolidateBlocks,` the collected runs by those
+      passes from the property set are used and there is no change in behavior
+      for the pass. The pass now runs up to 3x faster than it did with Python.
```

Comment:
Except for the last sentence this should be a separate note like it was before. This was documenting the new feature of the pass to run standalone without any of the collection passes.